show/main.py

```diff
@@ -977,6 +977,12 @@ def version(verbose):
     click.echo("ASIC: {}".format(asic_type))
     click.echo("ASIC Count: {}".format(asic_count))
     click.echo("Serial Number: {}".format(serial_number.stdout.read().strip()))
+    try:
+        sonic_platform.cli.show.version(verbose)
```

Comment:
is the `sonic_platform` module part of sonic-utilities repo? 
If other platforms also need to be dump platform specific information also how that can be achieved

Reply:
as per review comments, the code is restructured to the below: 

1) the underlying show_platform/cli/show.py will be moved under the respective plugin path as follows: 

          sonic-utilities/show/plugins/cisco-8000.py 

Similar future enhancements for config and clear options will be as: 

          sonic-utilities/config/plugins/cisco-8000.py
          sonic-utilities/clear/plugins/cisco-8000.py